.github/workflows/tests.yml

```diff
@@ -179,7 +179,7 @@ jobs:
   regressions:
     needs: process-results
     # Always check for regressions, as this can be skipped even if an indirect dependency fails (like a test run)
-    if: always()
+    if: always() && github.event_name != 'pull_request'
```

Comment:
I think that for the time being we should disable these checks on PRs. There is just too much variability to trust single measurements.

Reply:
Other option would be to push the number of standard deviations to 4 (currently at 3) so we will only see reports for crazy spikes.

Reply:
We already see huge spikes (well beyond any reasonable number of standard deviations). They aren't especially common, but if we run 100 tests and there is a one percent chance of a spike, that means a red CI for PRs most of the time.

I still don't think we have a good understanding of *why* the spikes happen, but I don't think checking for regressions on PRs is reliable enough due to them.